martor/settings.py

```diff
@@ -85,24 +85,30 @@
 )
 
 # Markdown urls
-MARTOR_UPLOAD_URL = getattr(
-    settings, "MARTOR_UPLOAD_URL", "/martor/uploader/"  # for imgur
+MARTOR_UPLOAD_URL = (
+    # Allows to disable this endpoint
+    settings.MARTOR_UPLOAD_URL if hasattr(settings, "MARTOR_UPLOAD_URL")
+    else "/martor/uploader/"
```

Comment:
This allows to set a falsy value in `settings.py` that will be respected by martor's config (no default set).